### ISSUE TITLE
Enable worker jobs to be added to job-type-specific queues

### ISSUE THREAD

**Issue by bbranan** [samvera-labs/hyku#1209]
Jobs which utilize external tools like FITS and ImageMagick tend to consume significant resources and can take a long time to complete relative to other job types.

In order to have worker nodes which are configured to best handle specific job types, it is necessary for those jobs to be distributed to different (SQS) queues. Each queue can then be matched with a properly sized worker pool.

Once this work is done at the application level, another ticket should be created to update the AWS stack in order to define multiple queues and configure distinct worker pools.

**Comment by atz** [comment]
The AWS provisioning is by far the harder piece of this.  I did a lot of work around queue configurability early on, but we abandoned that direction because of the difficulty of (dynamically) provisioning multiple queues.

**Comment by bbranan** [comment]
@atz I don't expect there to be dynamic queue creation here. I was anticipating a limited set of queues that are created as part of the initial stack. The number of queues is dependent on the number of job types that are likely to need instances with different configuration parameters. This may just be 2, one for simple jobs with low overhead and one for jobs which consume significant resources. When @eefahy and I were looking at this, the primary need seemed to be pulling FITS and Imagemagick jobs off of the primary queue and providing an alternative queue which is handled by much larger instance types.
